llama_index/vector_stores/pinecone.py

```diff
@@ -375,6 +383,9 @@ def query(self, query: VectorStoreQuery, **kwargs: Any) -> VectorStoreQueryResul
             if query.alpha is not None:
                 query_embedding = [v * query.alpha for v in query_embedding]
 
+            if not query_embedding and self._default_empty_query_vector is not None:
+                query_embedding = self._default_empty_query_vector
```

Comment:
what we can pass for `_default_empty_query_vector`? a not None but empty vector?